lib/signers/presign.js

```diff
@@ -11,11 +11,13 @@ var expiresHeader = 'presigned-expires';
  */
 function signedUrlBuilder(request) {
   var expires = request.httpRequest.headers[expiresHeader];
+  var isPresigner = true;
```

Comment:
Since the `isPresigner` variable is no longer used, it can be removed. (Same as a comment I made earlier, but I realized the other comment was made on an outdated diff)